README.md

```diff
@@ -1 +1,193 @@
-# CryptoLens
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# [cryptoLens: Your Real-Time Crypto Companion 🚀] <a name="about-project"></a>
+[Live Demo](https://www.loom.com/share/009df7a5d7584174926d23c016589e3f?sid=92e517cf-7224-4440-9f74-bf0b526c0568)
```

Comment:
- [ ] This video is only `45sec` and requirements suggest `a 5min` video please record another video
- containing a clear and objective description of the project and The topic and purpose of the project should be made clear
- containing a demo of the project, highlighting all the project features
- containing a highlight about a piece of the code you find interesting or you are proud of making